pkg/native/serializer.go

```diff
@@ -1,13 +1,21 @@
 package native
 
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -generate
+
 import (
 	"io"
 
 	"github.com/bom-squad/protobom/pkg/sbom"
-	"github.com/bom-squad/protobom/pkg/writer/options"
 )
 
+//counterfeiter:generate . Serializer
 type Serializer interface {
```

Comment:
@puerco @houdini91 in light of our discussion regarding interface pollution, I honestly think this file/interface belongs in either "writer" (the "user" of the serializer) or in a "domain" package. 

Serializers could implement the interface with ref to writers package.

`var _ native.Serializer = &CDX{}`
to 
`var _ writer.Serializer = &CDX{}`
making the beta clearer
`var _ writer.Serializer = &SPDX3{}`

Interface pollution - users is responsible for its interface, the implementation doesn't care (but it sometimes should, this is why we verify it does :P )